alchemy/src/state/file-system-state-store.ts

```diff
@@ -127,6 +127,30 @@ export class FileSystemStateStore implements StateStore {
     );
   }
 
+  async listStages(): Promise<string[]> {
+    const stages = new Set<string>();
+
+    try {
+      const rootStateDir = path.join(process.cwd(), ".alchemy");
+      const entries = await fs.promises.readdir(rootStateDir, {
+        withFileTypes: true,
+      });
```

Comment:
this feels inherintly flawed since we store other things there (logs, miniflare data, etc)

Reply:
That does look bad, yeah. State store is unfortunately legacy and really should be in its own folder under `/.alchemy` and not the root. Not sure how to achieve that for current users though.